cmd/juju/commands/debuglog.go

```diff
@@ -163,16 +163,16 @@ func (c *debugLogCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.Var(cmd.NewAppendStringsValue(&c.params.IncludeLabel), "include-label", "Only show log messages for these logging labels")
 	f.Var(cmd.NewAppendStringsValue(&c.params.ExcludeLabel), "exclude-label", "Do not show log messages for these logging labels")
 
-	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]")
+	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]. Logs of higher severity (e.g. ERROR) will still be shown when a lower level (e.g. INFO) is selected")
 	f.StringVar(&c.level, "level", "", "")
 
-	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines, and continue to append")
```

Comment:
Why remove "possibly filtered"?

Reply:
Hi Ian, I was discussing this with @SimonRichardson yesterday—'possibly filtered' seems redundant as it doesn’t specify what was filtered, possibly makes it more confusing to first time users as well in my opinion. Do you think we should include it?